ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GODictionaryRenderer/GODictionaryRenderer.cpp

```diff
@@ -67,12 +71,16 @@ void GODictionaryRenderer::componentComplete()
   // Create a map and give it to the MapView
   m_mapView = findChild<MapQuickView*>("mapView");
   m_map = new Map(Basemap::topographic(this), this);
-  m_mapView->setMap(m_map);
-  m_mapView->graphicsOverlays()->append(m_graphicsOverlay);
 
   parseXmlFile();
-  emit graphicsLoaded();
-  zoomToGraphics();
+  m_mapView->graphicsOverlays()->append(m_graphicsOverlay);
+
+  m_mapView->setMap(m_map);
+
+  connect(m_map, &Map::doneLoading, this, [this]()
```

Comment:
we might want to connect before setting the map on the mapview, which will trigger an implicit load. otherwise it is possible that it loads before the connection is made